src/borg_import/main.py

```diff
@@ -25,14 +25,19 @@ def borg_import(args, archive_name, path, timestamp=None):
 
     log.debug('Borg command line: %r', borg_cmdline)
     log.debug('Borg working directory: %s', path)
-    subprocess.check_call(borg_cmdline, cwd=str(path))
+    try:
+        subprocess.check_call(borg_cmdline, cwd=str(path))
+    except subprocess.CalledProcessError as cpe:
+        if cpe.returncode != 1:
+            raise
+        log.debug('Borg exited with a warning (being quiet about it since Borg spoke already)')
```

Comment:
somewhere at the beginning, we need to add "all your data will get assimilated now...". :-)